spec/units/builders/digest_email_builder_spec.rb

```diff
@@ -50,7 +50,23 @@
       title: "Test title 2"
     ).and_return("unsubscribe_link_2")
 
-    expect(ContentChangePresenter).to receive(:call).exactly(6).times
+    first_content_changes = subscription_content_change_results
+      .first
+      .content_changes
+
+    second_content_changes = subscription_content_change_results
+      .second
+      .content_changes
+
+    expect(ContentChangeDeduplicatorService).to receive(:call)
```

Comment:
I wonder if we can extract these two calls into a method to make it slightly clearer about what it does?

Reply:
fixed in 09c3dec